Sources/Mantis/Helpers/TransformStack.swift

```diff
@@ -0,0 +1,56 @@
+import UIKit
+
+
+class TransformStack: NSObject {
+    
+    static var shared: TransformStack = TransformStack()
+    
+    weak var transformDelegate : TransformDelegate?
+    
+    private var transformAdjustmentsStack: [TransformRecord] = []
+    
+    private var bottom: Int = 0
+    
+    var top: Int = 0
+    
+    private var transformStackBottom: Int = 0
+    private var transformStackTop: Int = 0
```

Comment:
`transformStackBottom` and `transformStackTop` are not used anywhere

Reply:
changes made